wp/wp-content/themes/lahma_maker/js/scripts.js

```diff
@@ -100,6 +100,15 @@ function arcsiStop(episodeName) {
     });
 }
 
+function arcsiDownload(episodeName, downloadLink) {
+    gtag('event', 'Arcsi download', {
+        'event_category': episodeName,
+        'event_label': 'Download name',
```

Comment:
Shouldn't `'Download name'` be `downloadLink` (the input parameter of the function)?

Reply:
No, 'Download name' is the label that is sent to gtag, downloadLink is the link to the show itself
line 109
`    window.open(downloadLink)
`

Reply:
Oh I see.